lib/utils.js

```diff
@@ -238,4 +378,6 @@ module.exports = {
   time,
   config,
   format,
+  fill,
```

Comment:
As these do not have general functionality across all files, they should be in `content/utils`